db.go

```diff
@@ -31,6 +31,14 @@ const (
 	GetIdentitiesFlagDeleted GetIdentitiesFlag = 1 << (iota - 1)
 )
 
+type AuthCheck int
+
+const (
+	AuthCheckNone             AuthCheck = 0
+	AuthCheckNonPasswordReuse AuthCheck = 1 << (iota - 1)
```

Comment:
If `AuthCheckNonPasswordReuse` could be turned into a positive flag, aka `AuthCheckPasswordReuse`, that would be cleaner